source/clear-linux/get-started/virtual-machine-install/vmware-esxi-preconfigured-cl-image.rst

```diff
@@ -32,7 +32,12 @@ Download the latest Clear Linux VMware image
 ********************************************
 
 Get the latest |CL| VMware image from the `image`_ repository.
-Look for :file:`clear-[version number]-vmware.vmdk.xz`.
+Look for :file:`clear-[version number]-vmware.vmdk.xz`. You can also use
+this command: 
+
+.. code-block:: bash
+
+   curl -O https://download.clearlinux.org/image/clear-$(curl https://download.clearlinux.org/latest)-vmware-vmdk.xz
```

Comment:
Oops!  My bad.  I gave you the wrong URL for Vmware.   It should be "curl -O https://download.clearlinux.org/image/clear-$(curl https://download.clearlinux.org/latest)-vmware.vmdk.xz"

Reply:
Yes. File extension will be changed from "...vmare-vmdk.xz" to "...vmare.vmdk.xz"